spatialprofilingtoolbox/db/importance_score_transcriber.py

```diff
@@ -48,23 +26,45 @@ def transcribe_importance(
         cohort_stratifier: str
             Name of the classification cohort variable the GNN was trained on to produce
                 the importance score.
-        database_connection_maker: DatabaseConnectionMaker
+        connection: psycopg2.connection
```

Comment:
Reference to module is more clear, but inconsistent with import. Should we adopt this style in the future?